erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -41,7 +41,7 @@ exports.ErizoJSController = function () {
     /*
      * Enables/Disables slideshow mode for a subscriber
      */
-    setSlideShow = function (slideShowMode, from, to){
+    setSlideShow = function (slideShowMode, from, to) {
```

Comment:
I think minor cosmetic problems like this in the future should be grouped in another PR, they add too much noise and create  merge conflicts when there shouldn't be.